### DISCUSSION TITLE
store anything (put any ship equipment into storage)

### DISCUSSION THREAD

**Discussion by realSanchez139** [winstxnhdw/lc-hax#516] [Q&A]
just wondering if there's a mod to do so from client side or if its host only

**Comment by winstxnhdw** [comment]
I don’t get it? What’s ship equipment? Items? Buildings? Clearly, we can already put buildings into storage but not items. There’s probably a way to store items in some ‘fake’ storage but only you would be able to access and retrieve it.

**Reply by realSanchez139** [reply]
yeah most likely but i used the /build command which basically got the result i wanted
